sgl-kernel/csrc/gemm/fp8_gemm_kernel.cu

```diff
@@ -833,6 +1113,17 @@ torch::Tensor fp8_scaled_mm(
 
   auto sm_version = getSMVersion();
 
+#if defined CUDA_VERSION && CUDA_VERSION >= 12080
+  if (sm_version == 100) {
```

Comment:
do we know if sm_version >= 100 would work? if the kernel doesn't require sm100a, we should be able to relax it.

Reply:
Yes. It support >=sm100.